src/common/tusb_mcu.h

```diff
@@ -356,6 +356,7 @@
 #elif TU_CHECK_MCU(OPT_MCU_RX63X, OPT_MCU_RX65X, OPT_MCU_RX72N, OPT_MCU_RAXXX)
   #define TUP_USBIP_RUSB2
   #define TUP_DCD_ENDPOINT_MAX    10
+  #define TUP_RHPORT_HIGHSPEED    1
```

Comment:
You need to put this statement inside conditional check as other MCUs don't support HS. For example `#if defined(BSP_MCU_GROUP_RA8M1)`

Reply:
OK